static/docs/command-reference/add.md

```diff
@@ -164,41 +164,45 @@ $ file .dvc/cache/d8/acabbfd4ee51c95da5d7628c7ef74b
 .dvc/cache/d8/acabbfd4ee51c95da5d7628c7ef74b: ASCII text
 ```
 
+Note that tracking compressed files (e.g. ZIP or TAR archives) is not
+recommended, as `dvc add` supports tracking directories. (Details below.) For
+more context, refer to
+[Data Registry](/doc/use-cases/data-registry#problem-1-compressed-data-files)
+
 ## Example: Directory
 
-What if you have not one dog picture, but hundreds of pictures of dogs and cats?
-Your goal might be to build an algorithm to identify dogs and cats in pictures,
-and this is your training dataset:
+Let's suppose your goal is to build an algorithm to identify cats and dogs in
+pictures. You may then have hundreds or thousands of pictures of these animals
+in a directory, and this is your training dataset:
 
 ```dvc
 $ tree pics
 pics
 ├── train
-│   ├── cats        <-- a lot of images of cats
-│   └── dogs        <-- a lot of images of dogs
+│   ├── cats        <-- A lot of images of cats
+│   └── dogs        <-- A lot of images of dogs
 └── validation
-    ├── cats        <-- images of cats
-    └── dogs        <-- images of dogs
+    ├── cats        <-- More images of cats
+    └── dogs        <-- More images of dogs
 ```
 
-Taking a directory under DVC control as simple as taking a single file:
+Taking a directory under DVC control as simple as with a single file:
 
 ```dvc
 $ dvc add pics
-
-Computing md5 for a large directory pics/train/cats. This is only done once.
+Computing md5 for a large number of files. This is only done once.
```

Comment:
This is the line I was referring to in https://github.com/iterative/dvc.org/pull/679#discussion_r336149908.